examples/tictactoe/src/components/Game.tsx

```diff
@@ -0,0 +1,247 @@
+        </span>
+
+        <p style={styles.infoTitle}>Opponent:</p>
+        <span style={{display: 'flex', alignItems: 'center', gap: '6px'}}>
+          <ChainLogo chainId={BigInt(game.opponentChainId!)}/>
+          <p style={styles.infoValue}>{truncateAddress(game.opponent!)}</p>
+        </span>
+      </div>
+      <Board game={game} />
+    </div>
+  )
+}
+
+const X: React.FC<{color?: string}> = ({color = '#3374DB'}) => {
+  return (
+    <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M14.9996 11.7018L26.5492 0.152344L29.849 3.45217L18.2994 15.0016L29.849 26.5509L26.5492 29.8507L14.9996 18.3014L3.45024 29.8507L0.150391 26.5509L11.6998 15.0016L0.150391 3.45217L3.45024 0.152344L14.9996 11.7018Z" fill={color}/>
+    </svg>
+  )
+}
+
+const O: React.FC<{color?: string}> = ({color = '#636779'}) => {
+  return (
+    <svg width="54" height="54" viewBox="0 0 54 54" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M27.0007 53.6667C12.2731 53.6667 0.333984 41.7275 0.333984 27C0.333984 12.2724 12.2731 0.333374 27.0007 0.333374C41.7281 0.333374 53.6673 12.2724 53.6673 27C53.6673 41.7275 41.7281 53.6667 27.0007 53.6667ZM27.0007 48.3334C38.7828 48.3334 48.334 38.7822 48.334 27C48.334 15.218 38.7828 5.66671 27.0007 5.66671C15.2186 5.66671 5.66732 15.218 5.66732 27C5.66732 38.7822 15.2186 48.3334 27.0007 48.3334Z" fill={color}/>
+    </svg>
+  )
+}
+
+const Board: React.FC<GameProps> = ({ game }) => {
+  const { chainId } = useAccount()
+  const { games } = useGames()
+  const { makeMove, isConfirming, isPending } = useMakeMove()
+  const { switchChain } = useSwitchChain() // TODO: Detect if required chain is not in the list
+
+  const [hoveredSquare, setHoveredSquare] = useState<string | null>(null);
+
+  const statusText = (status: GameStatus, turn: PlayerTurn) => {
+    switch (status) {
+      case GameStatus.Draw: return '🥱 Draw';
+      case GameStatus.Won: return '🥳 You won';
+      case GameStatus.Lost: return '👎 You lost';
+      default: // game is active
+    }
+
+    if (!game.opponent) { return '-' }
+
+    switch (turn) {
+      case PlayerTurn.Player: return '🖐️ Your turn';
+      case PlayerTurn.Opponent: return "‍💻 Opponent's turn";
+      default: return 'UNEXPECTED';
+    }
+  };
+
+  const isPlayerTurn = game.turn === PlayerTurn.Player
+  const isConnectedToChain = chainId === Number(game.lastActionId.chainId)
+  const isGameOver = game.status !== GameStatus.Active
+
+  const gameEndingMoves = endingMoves(game.moves)
+  const cellBackgroundColor = (row: number, col: number) => {
+    if (!gameEndingMoves) { return '#F2F3F8' }
+
+    const move = game.moves[row][col]
+
+    // winning cell
+    if (move == 1 && gameEndingMoves.some(move => move[0] === row && move[1] === col)) { return '#D6FFDA' }
+
+    // losing cell
+    if (move == 2 && gameEndingMoves.some(move => move[0] === row && move[1] === col)) { return '#FFD6D6' }
+
+    return '#F2F3F8'
+  }
```

Comment:
it would be better to do this type of logic with classnames

Reply:
agreed, this will be apart of the cleanup